src/lib/crypto/kimchi_bindings/stubs/Cargo.toml

```diff
@@ -12,32 +12,43 @@ name = "wires_15_stubs"
 crate-type = ["lib", "staticlib"]
 
 [dependencies]
-array-init = "2.0.0"
-rmp-serde = "1.1.2"
-libc = "0.2.62"
-num-bigint = { version = "0.4.4", features = [ "rand", "serde" ] }
-paste = "1.0.5"
-rand = "0.8.5"
-rayon = "1.5.0"
-serde = "1.0.130"
-serde_json = "1.0.103"
-sprs = { version = "0.11.0", features = ["multi_thread"] }
-once_cell = "1.10.0"
+array-init = { workspace = true }
+libc = { workspace = true }
+num-bigint = { workspace = true, features = [ "rand", "serde" ] }
+# ocaml-specific
+ocaml = { workspace = true, features = ["no-caml-startup"] }
+ocaml-gen = { workspace = true }
+once_cell = { workspace = true }
+paste = { workspace = true }
+rand = { workspace = true }
+rayon = { workspace = true }
+rmp-serde = { workspace = true }
+serde = { workspace = true }
+serde_json = { workspace = true }
+sprs = { workspace = true, features = ["multi_thread"] }
 
 # arkworks
-ark-ff = { version = "0.4.2", features = ["parallel", "asm"] }
-ark-serialize = "0.4.2"
-ark-ec = { version = "0.4.2", features = ["parallel"] }
-ark-poly = { version = "0.4.2", features = ["parallel"] }
+ark-ec = { workspace = true, features = ["parallel"] }
+ark-ff = { workspace = true, features = ["parallel", "asm"] }
+ark-poly = { workspace = true, features = ["parallel"] }
+ark-serialize = { workspace = true }
 
 # proof-systems
-poly-commitment = { path = "../../proof-systems/poly-commitment", features = ["ocaml_types"] }
-groupmap = { path = "../../proof-systems/groupmap" }
-mina-curves = { path = "../../proof-systems/curves" }
-o1-utils = { path = "../../proof-systems/utils" }
-mina-poseidon = { path = "../../proof-systems/poseidon" }
-kimchi = { path = "../../proof-systems/kimchi", features = ["ocaml_types"] }
+groupmap = { workspace = true }
+kimchi = { workspace = true, features = ["ocaml_types"] }
+mina-curves = { workspace = true }
+mina-poseidon = { workspace = true }
+o1-utils = { workspace = true }
+poly-commitment = { workspace = true, features = ["ocaml_types"] }
 
-# ocaml-specific
-ocaml = { version = "0.22.2", features = ["no-caml-startup"] }
-ocaml-gen = "0.1.5"
+# Temporary transitive dependencies
+# Remove as soon as possible. It is temporary to get Rust 1.72 working.
+## To espace this error:
+## ```
+## package `bumpalo v3.16.0` cannot be built because it
+## requires rustc 1.73.0 or newer, while the currently
+## active rustc version is 1.72.1
+## ```
+bumpalo = { version = "=3.14.0" }
```

Comment:
Both `bumpalo` and `cc` versions here match the ones in the workspace, why not use `.workspace = true` as with other dependencies?